eZ/Publish/Core/Search/Legacy/Content/Common/Gateway/CriterionHandler/FieldValue/Handler/Keyword.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * File containing the DoctrineDatabase Composite field value handler class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Search\Legacy\Content\Common\Gateway\CriterionHandler\FieldValue\Handler;
+
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\Core\Persistence\Database\SelectQuery;
+
+class Keyword extends Collection
```

Comment:
Please add PhpDoc briefly stating something like "FieldValue CriterionHandler handling ezkeyword External Storage for Legacy/SQL Search".